### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by pabrcno**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,38 @@
-# Privacy Policy for Topics
+Sure, below is your updated Privacy Policy including the new permissions:
```
Comment (diegoasanch) [inline README.md]: This seems like an llm response. Maybe forgot to remove it? 🤖

File: README.md [hunk 2]
```diff
@@ -1,37 +1,38 @@
...
 
-Storage: Our app requires access to your device's storage to store internal state related to multiple versions. This helps us maintain your preferences and provide a consistent user experience.
+**Storage:** Our app requires access to your device's storage to store internal state related to multiple versions. This helps us maintain your preferences and provide a consistent user experience.
 
-User Data: We store user data in Firebase, adhering to strict authentication policies to ensure the security and privacy of your data. This data is used to personalize your experience and provide you with relevant features and services.
+**Microphone and Internet:** Our app requires access to your device's microphone and internet connection to facilitate the voice and audio related functionalities. This data is not stored or transmitted and is used exclusively for providing the stated functionalities within the app.
 
-## How We Use Your Information
+**Bluetooth:** The app requires Bluetooth access for potential future features related to device communication and interaction. No data is extracted from your Bluetooth devices by our app.
```
Comment (diegoasanch) [inline README.md:14]: Why require permissions for a service that's not implement yet? Wouldn't it be better to wait unit something needs the Bluetooth connectivity to ask?

**Review by diegoasanch — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (pabrcno) [reply README.md]: SEP

Comment (pabrcno) [reply README.md]: DONE

File: README.md (hunk 2 above)
Comment (pabrcno) [reply README.md:14]: https://github.com/pabrcno/topics/blob/main/android/app/src/main/AndroidManifest.xml

Utilizamos bt para los text to speech functionality, it is needed for headphone connection

Comment (diegoasanch) [reply README.md:14]: Then should that be mentioned here?

**Pull request merged**